README.md

```diff
@@ -36,7 +36,32 @@
 
 ## Usage
 
-Install via the [Chrome Web Store](https://chromewebstore.google.com/detail/catppuccin-for-github-fil/lnjaiaapbakfhlbjenjkhffcdpoompki) or [Firefox Addons](https://addons.mozilla.org/en-US/firefox/addon/catppuccin-gh-file-explorer/).
+<p align="center">
```

Comment:
Maybe something like

```suggestion
The extension is available on Chrome, Firefox and Microsoft Edge extension stores. Click on your browser's badge to install:

<p align="center">
```

It might not be intuitive for people that they have to click on the badges. Feel free to reword my suggestion but just some text above it would be good imo.